internal/app/commands/common/common.go

```diff
@@ -59,7 +59,7 @@ var AUTH_FLAG = cli.StringFlag{
 
 var CRED_FILE_FLAG = cli.StringFlag{
 	Name:  "cred-file",
-	Usage: "The absolute path to the service account key data file (the file must be in JSON format)",
+	Usage: "The absolute path to the service account key file (in JSON format). This file is used to create a signed JWT token, which is included in the \"Authorization: Bearer <token>\" header to access protected resources. This is available only for users with System ID Provider. If specified, the flag \"--auth\" or \"-a\" will be ignored",
```

Comment:
Authorization header - is an implementation detail, no need to be so specific.
```suggestion
	Usage: "The absolute path to the service account key file (in JSON format). This is only available for XP version 7.15 and later. Key file can be generated by Users application for System ID Provider users (aka Service Accounts) . If specified, the flag \"--auth\" or \"-a\" will be ignored",
```